stl/msbuild/stl_2/xmd/dirs.proj

```diff
@@ -6,9 +6,7 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
     <Import Project="$(MSBuildThisFileDirectory)..\..\..\..\..\crt-common.settings.targets" />
 
-    <ItemGroup Condition="('$(SpectreBuildMode)' == '' or
-                           '$(Configuration)' == 'chk' or
-                           '$(Configuration)' == 'dbg')">
+    <ItemGroup>
         <ProjectFile Include="msvcp_2_xmd_app.vcxproj" />
         <ProjectFile Include="msvcp_2_xmd_kernel32.vcxproj" />
         <ProjectFile Include="msvcp_2_xmd_onecore.vcxproj" />

```

Comment:
(Also commented in MSVC-PR-569343, expand context immediately below.)

I observe that 6 files are still guarding netfx projects for spectre. Is this intentional?

Reply:
Yep, this is only to update the binaries we ship in VS. CLR doesn't need spectre-mitigated debug libraries.